### ISSUE TITLE
build issues with netapi32-sys : use of experimental attribute

### ISSUE THREAD

**Issue by eminence** [retep998/winapi-rs#333]
Is this a known or expected issue?  I'm running `rustc 1.14.0-nightly (289f3a4ca 2016-09-29)`
```
   Compiling netapi32-sys v0.0.1
error: The attribute `experimental` is currently unknown to the compiler and may have meaning added to it in the future (see issue #29642)
 --> C:\Users\lxd\.cargo\registry\src\github.com-1ecc6299db9ec823\netapi32-sys-0.0.1\src\lib.rs:5:1
  |
5 | #![experimental]
  | ^^^^^^^^^^^^^^^^
  |
  = help: add #![feature(custom_attribute)] to the crate attributes to enable

error: aborting due to previous error

error: Could not compile `netapi32-sys`.

```

**Comment by retep998** [comment]
That crate is just an empty stub from a _very_ long time ago and will remain so due to the [winapi 0.3 transition](https://github.com/retep998/winapi-rs/issues/316).

**Comment by eminence** [comment]
It's not really empty, though:  https://github.com/retep998/winapi-rs/blob/dev/lib/netapi32/src/lib.rs

**Comment by retep998** [comment]
Yeah, there are some -sys crates that have definitions in them, but I never got around to publishing them. Anyway, I published `netapi32-sys` `0.2.0`, since the winapi 0.3 overhaul is taking such a long time.
